packages/server/src/startup.ts

```diff
@@ -58,7 +58,7 @@ export async function startup(app?: any, server?: any) {
     return
   }
   STARTUP_RAN = true
-  if (server) {
+  if (server && env.CLUSTER_MODE?.toLowerCase() !== "true") {
```

Comment:
You could just do `&& env.CLUSTER_MODE` here I believe. Usually for env variables we just set them to `1` if it's a boolean

Reply:
changed it accordingly! Let me know if that works